packages/core/src/common/utils/isDarkTheme.ts

```diff
@@ -0,0 +1,12 @@
+/*
```

Comment:
Let's rename this file to `themeUtils.ts` (bags of functions are always called `___Utils.ts` in our codebase).

Reply:
disagree! I've come to greatly prefer the pattern where you have a `utils/` _directory_ and a bunch of individual files, just like this.

Reply:
I don't particularly want to have two schemes in play at the same time in the codebase, but okay.